no message

KonishiNorifumi opened the pull request.

KonishiNorifumi committed:
no message

manaberuit commented on input.php:
```diff
@@ -0,0 +1,63 @@
+     $num1 = $_POST["fizz"];
+     $num2 = $_POST["buzz"];
+
+ if(strval($num1) === strval(intval($num1)) && (strval($num2)==strval(intval($num2))))
```
こちらの処理は何のための処理でしょうか？

manaberuit commented on input.php:
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```
全体的にインデントがずれていますので、こちら修正してください。

KonishiNorifumi replied:
修正します！

manaberuit commented on input.php:
```diff
@@ -0,0 +1,63 @@
+      break;}
+
+
+      if($ans1 == $ans2) {
```
==ではなく、===を使用しましょう！
==は自動で型変換が行われてしまいバグの原因になりやすいので、業務では基本使用されません。

manaberuit reviewed (commented):
３点コメントしましたので、ご確認ください。

KonishiNorifumi committed:
修正点
全体的なインデントの修正
"=="から"==="へ
もう一点、ご指摘頂いていた処理ですが、整数判定のために用いております。（is_intやis_numericでは正しく動作しなかったため）

manaberuit reviewed (approved):
OKです^^
